nevergrad/optimization/test_optimizerlib.py

```diff
@@ -484,6 +484,55 @@ def test_bo_ordering() -> None:
     optim.provide_recommendation()
 
 
+@pytest.mark.parametrize(  # type: ignore
```

Comment:
I think I would prefer seeing this in a test_nghyperopt.py file, because it's quite specific and complex, because more tests may be added eventually to test specifically the helper functions (_parametrization_to_dict etc), and because eventually I need to split this test file which is getting always bigger :D